examples/gke-consumption-options/dws-flex-start-queued-provisioning/gke-a3-ultragpu.yaml

```diff
@@ -33,7 +33,7 @@ vars:
   # Installs NCCL library and Google NCCL plugin
   # Runs an init container on all H200 GPU nodes with the NCCL plugin image
   nccl_installer_path: $(ghpc_stage("./nccl-installer.yaml"))
-  static_gpu_count: 512
+  kueue_gpu_nomial_quota: 512
```

Comment:
kueue_gpu_nomial_quota should be `kueue_gpu_nominal_quota`

Reply:
Let's rename to a more generic `gpu_nominal_quota` for now. We can make it specific if there's a need in the future